Makefile

```diff
@@ -114,8 +120,18 @@ rules: $(GAME)/$(GAME)
 	 ./$(GAME) genrules $(GAME)_intro.html $(GAME).css html/$(GAME).html \
 	)
 
+.PHONY: unittest
+unittest:
+	$(MAKE) GAME=unittest unittest-build
+
```

Comment:
Because the unittest needs a slightly different set of objects, it was easier just to have a new target rather than mucking with the $(GAME)/$(GAME) target to handle this oddball case.  But due to the fact that we need to game objects from *some* game in order to compile fully, having unittest be it's own (very limited) game with no ruleset/world/etc functionality made sense.